spec/lib/percy/capybara/client/builds_spec.rb

```diff
@@ -31,25 +33,25 @@
           'id' => '123',
           'type' => 'builds',
           'relationships' => {
-            'self' => "/api/v1/snapshots/123",
+            'self' => '/api/v1/snapshots/123',
             'missing-resources' => {
               'data' => [
                 {
                   'type' => 'resources',
-                  'id' => loader.build_resources.first.sha,
-                },
-              ],
-            },
-          },
-        },
+                  'id' => loader.build_resources.first.sha
+                }
+              ]
+            }
+          }
+        }
       }
       # Stub create build.
       build_stub = stub_request(:post, builds_api_url)
-        .to_return(status: 201, body: mock_response.to_json)
+                   .to_return(status: 201, body: mock_response.to_json)
```

Comment:
Same here as other comment (but maybe a different rubocop config), I'd like to enforce a 2-space (or 4-space, arguably) indent for continued line-break lines, and not align them to the line above.